src/Service/Azure.DataApiBuilder.Service.csproj

```diff
@@ -75,7 +72,7 @@
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
         <PackageReference Include="System.IO.Abstractions" />
-
+        <PackageReference Include="Microsoft.SourceLink.GitHub" PrivateAssets="All"/>
```

Comment:
Just curious, do we package test project binaries?  I guess not, is that why we dont need this addition in test project?